pkg/migrations/sql/idempotent/base.sql

```diff
@@ -1253,18 +1253,18 @@ COMMENT ON FUNCTION SCHEMA_PROM.reset_metric_chunk_interval(TEXT)
 IS 'resets the chunk interval for a specific metric to using the default';
 GRANT EXECUTE ON FUNCTION SCHEMA_PROM.reset_metric_chunk_interval(TEXT) TO prom_admin;
 
-CREATE OR REPLACE FUNCTION SCHEMA_CATALOG.get_metric_retention_period(metric_name TEXT)
+CREATE OR REPLACE FUNCTION SCHEMA_CATALOG.get_metric_retention_period(schema_name TEXT, metric_name TEXT)
```

Comment:
This isn't a great interface change TBH from the UIUX point of view. Maybe this should be the second parameter but with a default value of 'prom_data'. I know that violates the argument ordering though. I think this needs a larger discussion.